graph-refresh/base/cronjob.yaml

```diff
@@ -0,0 +1,132 @@
+---
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: graph-refresh
+  labels:
+    app: thoth
+    component: graph-refresh
+spec:
+  schedule: "0 * * * *"
+  suspend: true
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
+  startingDeadlineSeconds: null
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: graph-refresh
+        spec:
+          serviceAccountName: "graph-refresh-job"
+          containers:
+            - image: "graph-refresh-job:latest"
+              name: graph-refresh
+              env:
+                - name: "THOTH_SOLVER_OUTPUT"
+                  # TODO: move these hardcoded values of solver output into cm
+                  value: http://result-api/api/v1/solver-result
```

Comment:
This can be removed, we do not need solver output anymore.

Reply:
ack, thanks for pointing this.